Source/WebCore/css/parser/CSSPropertyParserHelpers.h

```diff
@@ -247,7 +247,7 @@ template<CSSValueID... names> RefPtr<CSSPrimitiveValue> consumeIdentWorkerSafe(C
 
 inline bool isFontStyleAngleInRange(double angleInDegrees)
 {
-    return angleInDegrees > -90 && angleInDegrees < 90;
+    return angleInDegrees >= -90 && angleInDegrees <= 90;
```

Comment:
You might want to update the changelog now that I made the change in a different commit.

Reply:
OK, will do.